.github/workflows/engine-checks.yml

```diff
@@ -209,6 +209,47 @@ jobs:
       GRAAL_EDITION: GraalVM CE
     permissions:
       checks: write
+  enso-build-ci-gen-job-standard-library-api-check-linux-amd64:
+    name: Standard Library API check (linux, amd64)
+    runs-on:
+      - self-hosted
+      - Linux
+    steps:
+      - if: runner.os == 'Windows'
```

Comment:
`runs-on: Linux` kinda implies this condition is going to be false.

Reply:
This is repeated step for every generated job in this file, and it other generated files. This is simply what you get when you run `cd build_tools/ci-gen; cargo run`. Not anything I can, or want, to change.